plugin/gatsby-node.js

```diff
@@ -4,5 +4,6 @@ exports.pluginOptionsSchema = ({ Joi }) => {
     colors: Joi.array()
       .items(Joi.string().default("#ffffff"))
       .description("Array of hex color values"),
+    intensity: Joi.string().valid('light-snow', 'blizzard'),
```

Comment:
**nitpick**: Let's rename "light-snow" to just "light"
**nitpick**: Let's rename "default" to "regular", make it a valid option and make it the default.

Reply:
Done ✅

> The "blizzard" is so much fun ❄️

Glad you like it 😄